UsacoLeeland/src/UsacoLeeland/BlockedBillboard.java

```diff
@@ -0,0 +1,114 @@
+	{
+		int dif=0;
+		int[][] cord=new int [3][4];
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		String temp=br.readLine();
+		sort(0,cord,temp.split("\\s+"));
+		temp=br.readLine();
+		sort(1,cord,temp.split("\\s+"));
+		temp=br.readLine();
+		sort(2,cord,temp.split("\\s+"));
+		for(int j=0; j<2; j++)
+		{
+			boolean[] res=new boolean[4];
+			int count=0;
+			if(cord[j][0]>cord[2][0]&&cord[j][0]<cord[2][2]&&cord[j][1]>cord[2][1]&&cord[j][1]<cord[2][3])
+				res[0]=true;
+			if(cord[j][2]>cord[2][0]&&cord[j][2]<cord[2][2]&&cord[j][3]>cord[2][1]&&cord[j][3]<cord[2][3])
+				res[1]=true;
+			if(cord[j][0]>cord[2][0]&&cord[j][0]<cord[2][2]&&cord[j][3]>cord[2][1]&&cord[j][3]<cord[2][3])
+				res[2]=true;
+			if(cord[j][2]>cord[2][0]&&cord[j][2]<cord[2][2]&&cord[j][1]>cord[2][1]&&cord[j][1]<cord[2][3])
+				res[3]=true;
+			for(int i=0; i<4; i++)
+				{
+					if(res[i])
+						count++;
+				}
+			int[] tem=new int[2];
+			int count1=0;
+			if(count==0)
+			{
+				if(cord[2][0]>cord[j][0]&&cord[2][0]<cord[j][2]&&cord[2][1]>cord[j][1]&&cord[2][1]<cord[j][3])
+					res[0]=true;
+				if(cord[2][2]>cord[j][0]&&cord[2][2]<cord[j][2]&&cord[2][3]>cord[j][1]&&cord[2][3]<cord[j][3])
+					res[1]=true;
+				if(cord[2][0]>cord[j][0]&&cord[2][0]<cord[j][2]&&cord[2][3]>cord[j][1]&&cord[2][3]<cord[j][3])
+					res[2]=true;
+				if(cord[2][2]>cord[j][0]&&cord[2][2]<cord[j][2]&&cord[2][1]>cord[j][1]&&cord[2][1]<cord[j][3])
+					res[3]=true;
+				for(int i=0; i<4; i++)
+				{
+					if(res[i])
+					{
+						tem[count1]=i;
+						count1++;
+					}
+				}
+				if(tem[0]==0&&tem[1]==2)
+					dif+=(cord[j][2]-cord[2][0])*(cord[2][3]-cord[2][1]);
+				if(tem[0]==0&&tem[1]==3)
+					dif+=(cord[j][3]-cord[2][1])*(cord[2][2]-cord[2][0]);
+				if(tem[0]==1&&tem[1]==2)
+					dif+=(cord[2][1]-cord[j][3])*(cord[2][2]-cord[2][0]);
+				if(tem[0]==1&&tem[1]==3)
+					dif+=(cord[2][2]-cord[j][0])*(cord[2][3]-cord[2][1]);
+			}
+			if(count==1)
+			{
+				for(int i=0; i<4; i++)
+				{
+					if(res[i])
+					{
+						if(i==0)
+							dif+=(cord[2][2]-cord[j][0])*(cord[2][3]-cord[j][1]);
+						if(i==1)
+							dif+=(cord[j][2]-cord[2][0])*(cord[j][3]-cord[2][1]);
+						if(i==2)
+							dif+=(cord[2][2]-cord[j][0])*(cord[j][3]-cord[2][1]);
+						if(i==3)
+							dif+=(cord[j][2]-cord[2][0])*(cord[2][3]-cord[j][1]);
+					}
+			
+				}
+			}
+			if(count==2)
+			{
+				for(int i=0; i<4; i++)
+				{
+					if(res[i])
+					{
+						tem[count1]=i;
+						count1++;
+					}
+				}
+				if(tem[0]==0&&tem[1]==2)
+					dif+=(cord[2][2]-cord[j][0])*(cord[j][3]-cord[j][1]);
+				if(tem[0]==0&&tem[1]==3)
+					dif+=(cord[2][3]-cord[j][1])*(cord[j][2]-cord[j][0]);
+				if(tem[0]==1&&tem[1]==2)
+					dif+=(cord[j][1]-cord[2][3])*(cord[j][2]-cord[j][0]);
+				if(tem[0]==1&&tem[1]==3)
+					dif+=(cord[j][0]-cord[2][2])*(cord[j][3]-cord[j][1]);
+			}
+			if(count==4)
+				dif+=(cord[j][2]-cord[j][0])*(cord[j][3]-cord[j][1]);
+		}

```

Comment:
Is there a way to make the code cleaner and more readable? The skill of writing readable code is very essential. Think about when you have a bug and you have no idea where the bug is -- you'll need read your code again and again trying to spot the mistake or omission. I emphasize the *again and again* part. So if you are not able to write readable code, that will be where you waste of your competition time.